src/display/canvas.js

```diff
@@ -1120,6 +1124,15 @@ class CanvasGraphics {
     this._cachedGetSinglePixelWidth = null;
   }
 
+  getObject(data, _default) {
```

Comment:
How about `_default` -> `fallback` instead? 

Also, I'd suggest changing this to `getObject(data, fallback = null) {` since some of the call-sites below would then not need to provide a second argument any more.